### PR TITLE
fix(osv/python): skip GHSA-IDs

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
Added code to skip GHSA-IDs when OSV database is processed

## Related issues
- aquasecurity/trivy-db/issues/179

### CODE REVIEW

File: pkg/vulnsrc/osv/osv_test.go [hunk 1]
```diff
@@ -108,3 +109,76 @@ func TestVulnSrc_Update(t *testing.T) {
...
+
+func TestVulnSrc_PythonGHSAIDsIsSkipped(t *testing.T) {
+	tests := []struct {
+		name          string
+		dir           string
+		pythonBuckets []wantKV
+		rustBuckets   []wantKV
+	}{
+		{
+			name: "happy path, when python GHSA-ID is skipped",
```
Comment (knqyf263) [inline pkg/vulnsrc/osv/osv_test.go:121]: I think we can merge this case to TestVulnSrc_Update.

File: pkg/vulnsrc/osv/osv.go [hunk 2]
```diff
@@ -82,6 +82,11 @@ func (vs VulnSrc) Update(dir string) error {
 func (vs VulnSrc) save(eco ecosystem, entries []Entry) error {
 	err := vs.dbc.BatchUpdate(func(tx *bolt.Tx) error {
 		for _, entry := range entries {
+			//GHSA-IDs are already stored via ghsa package.
```
Comment (knqyf263) [inline pkg/vulnsrc/osv/osv.go:85]: It should be done when decoding to JSON.

**Review by knqyf263 — CHANGES_REQUESTED** [review]

File: pkg/vulnsrc/osv/osv.go (hunk 2 above)
Comment (DmitriyLewen) [reply pkg/vulnsrc/osv/osv.go]: Moved

File: pkg/vulnsrc/osv/osv_test.go (hunk 1 above)
Comment (DmitriyLewen) [reply pkg/vulnsrc/osv/osv_test.go]: Done

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**